t04_refactored.py

```diff
@@ -137,6 +137,66 @@ def team_3_adv():
 def team_4_adv():
     pass
     # TODO Add your code here
+    # """
+    # https://docs.google.com/document/d/1BT_3NDURoZ_V-Ztx1OQcLPjn5krxuLOk8cLPmETCF2c/edit
+    # Kafui Gle
+    # Samundra Adhikari
+    # :return: none
+    # """
+from time import sleep
+
+delay = 1.0
```

Comment:
You shouldn't need either of these lines of code. Check the first few lines of code in the file. 

Also, all of your code should be inside your function. This is unindented, making it outside your function.

Reply:
issue has been addressed please.